### PR TITLE
Add docs for rules with past runs backfill

### PR DESCRIPTION
**Pull request by akira**

### CODE REVIEW

File: versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx [hunk 1]
```diff
@@ -43,11 +43,22 @@ You can create a filter as you normally would to filter traces in the project. F
 
 ![](../static/rules_filter.png)
 
-### Step 3: Define the sampling rate
+## (Optional) Step 3: Apply Rule to Past Runs
```
Comment (jakerachleff) [inline versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx:46]:
```suggestion
### (Optional) Step 3: Apply Rule to Past Runs
```

File: versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx [hunk 2]
```diff
@@ -43,11 +43,22 @@ You can create a filter as you normally would to filter traces in the project. F
...
+
+When creating a new rule, you can apply the rule to past runs as well. To do this, select **Apply to Past Runs** checkbox and enter **Backfill From** date as the
+start date to apply the rule.
+
+This will start from the **Backfill From** date and apply the run rules until it is caught up with the latest runs.
+
+![](../static/rules_past_runs.png)
+
+Note that you will have to expand the date range for logs if you wanted to look at the progress of the backfill, see [View logs for your automations](./how_to_guides/monitoring/rules#view-logs-for-your-automations) for details.
```
Comment (jakerachleff) [inline versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx:55]: This is broken - it should be something like ./#view-logs-for-your-automations but forget exact syntax

File: versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx (hunk 1 above)
Comment (akira) [reply versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx]: Updated

File: versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx (hunk 2 above)
Comment (akira) [reply versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx]: Ah ok good catch, ./rules/#... seems to work

**Review by jakerachleff — APPROVED** [review]

**Pull request merged**